# High Normal Low
string value
---
int32 errorCode
string errorDescription